SmartDeviceLink/SDLStreamingVideoLifecycleManager.m

```diff
@@ -112,11 +114,11 @@ - (instancetype)initWithConnectionManager:(id<SDLConnectionManagerType>)connecti
         }
 
         SDLLogD(@"Initializing CarWindow");
-        _carWindow = [[SDLCarWindow alloc] initWithStreamManager:self configuration:configuration.streamingMediaConfig];
+        _carWindow = [[SDLCarWindow alloc] initWithStreamManager:self configuration:configuration.streamingMediaConfig scale:self.sdl_scale];
```

Comment:
From the proposal, it looks like you also need to handle the frames for the haptic rectangles. Android does this in their PR: https://github.com/skibyte/sdl_java_suite/blob/a4aebd975862a572dc4bf3ff7d78c5710dfe7803/android/sdl_android/src/main/java/com/smartdevicelink/haptic/HapticInterfaceManager.java#L116

Reply:
done in commit 46ed0f04